pkg/glanceapi/statefulset.go

```diff
@@ -60,12 +61,13 @@ func StatefulSet(
 		PeriodSeconds:    10,
 	}
 	livenessProbe := &corev1.Probe{
-		PeriodSeconds:       3,
-		InitialDelaySeconds: 3,
+		TimeoutSeconds:      30,
+		PeriodSeconds:       30,
+		InitialDelaySeconds: 30,
```

Comment:
that seems to be excessive compared to other patches

Reply:
fixed